platform-sdk/consensus-otter-tests/src/testFixtures/java/org/hiero/otter/fixtures/container/ContainerNode.java

```diff
@@ -138,7 +138,22 @@ public AsyncNodeActions withTimeout(@NonNull final Duration timeout) {
      */
     @Override
     public void submitTransaction(@NonNull final byte[] transaction) {
-        throw new UnsupportedOperationException("Not implemented yet!");
+        throwIfIn(INIT, "Node has not been started yet.");
+        throwIfIn(SHUTDOWN, "Node has been shut down.");
+        throwIfIn(DESTROYED, "Node has been destroyed.");
+
+        try {
+            final TransactionRequest request = TransactionRequest.newBuilder()
+                    .setPayload(ByteString.copyFrom(transaction))
+                    .build();
+
+            final TransactionRequestAnswer answer = blockingStub.submitTransaction(request);
+            if (!answer.getResult()) {
+                fail("Failed to submit transaction for node %d.".formatted(selfId.id()));
+            }
+        } catch (final Exception e) {
```

Comment:
If this happens, we should probably fail the test or let the caller deal with the `Exception` (which will most likely also result in a failed test).